### PR TITLE
Support `make base CPUS=n`.

### PR DESCRIPTION
**Pull request by samth**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -151,6 +151,15 @@ CONFIGURE_ARGS_qq =
...
 
 base:
+	if [ "$(CPUS)" = "" ] ; \
+         then $(MAKE) plain-base ; \
+         else $(MAKE) cpus-base CPUS="$(CPUS)" ; fi
+
+cpus-base:
+	$(MAKE) -j $(CPUS) plain-base JOB_OPTIONS="-j $(CPUS)"
+
+
```
Comment (mflatt) [inline Makefile]: intentional extra blank line?

**Comment by mflatt** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (samth) [reply Makefile]: No, not intentional.

**Comment by samth** [comment]
Ok, I removed the blank line and will merge after CI completes.

**Pull request merged**